lib/routes/retell.js

```diff
@@ -53,6 +56,11 @@ const service = ({logger, makeService}) => {
             trunk: process.env.PSTN_TRUNK_NAME
           }
         ];
+        /* Workaround for SIPGATE, put User ID as from and CLI in Diversion header */
+        if (OVERRIDE_FROM_USER) {
+          headers.Diversion = from;
```

Comment:
Hmm, I thought the point was to put a "real" calling party number in the Diversion header so the person answering sees that as the caller id, while putting the sipgate sip-id in the from header

Reply:
as I understand it, sipgate gives you a "SIP-ID" which is not a phone number yet which needs to the in the From header.  However, I dont think that is what should appear as a caller id to the party receiving the call, and that is what I thought the Diversion header was for.  This is sort of like how Voxbone used to work if I recall correctly

Reply:
yes, but thats what this change will do, so you configure the app and retell all normally with your phone number
Then set the OVERIDE_FROM_USER env var to be your SIP-ID,
When a call comes from retell the from will be the phone number, this change will move that number to the Diversion header and set the from to be your SIP-ID